src/test/java/redis/clients/jedis/tests/UDSTest.java

```diff
@@ -0,0 +1,19 @@
+package redis.clients.jedis.tests;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.tests.HostAndPortUtil;
+
+public class UDSTest {
+  protected static String udsHost = HostAndPortUtil.getUDSServers().get(0);
```

Comment:
I used to write some more test cases, but one looks like is enough. Will fix it.